### ISSUE TITLE
clip_subset_type  GDAL is invalid - version 7.311

### ISSUE THREAD

**Issue by MichaelJendryke** [sahrk/DGGRID#40]
When running `DGGRID` with `clip_subset_type GDAL` I get the following error

```
** executing DGGRID version 7.3 **
type sizes: big int: 64 bits / big double: 128 bits

** loading meta file /root/metafile_17c60ecc-1f71-11ec-9b03-ac1f6bf4b0d4_res16.meta...
FATAL ERROR: Invalid parameter data in parameter:
clip_subset_type INVALID (N/A)
Value 'GDAL' is not one of the allowed values:
whole_earth
aigen
shapefile
seqnums
points
```

GDAL is present on the system, see `cmake` output below:
```
cmake -DCMAKE_BUILD_TYPE=Release ..
-- The C compiler identification is GNU 8.4.1
-- The CXX compiler identification is GNU 8.4.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Doxygen: /usr/bin/doxygen (found version "1.8.14") found components: doxygen dot
-- Found GDAL: /usr/lib64/libgdal.so (found version "3.0.4")
-- Configuring done
-- Generating done
-- Build files have been written to: /home/DGGRID/build
```

**Comment by MichaelJendryke** [comment]
I just tried it with version 7.1 and it works perfectly. Sorry if I opened an issue for a future release.

**Comment by sahrk** [comment]
You didn't. I just checked and GDAL using `cmake` looks broken for me as well. I'll have to backtrack and figure out what I screwed up. A work-around would be to build using the old makefiles, which are still there (see the text file`README.NOCMAKE`).

In case @r-barnes is reading this: it looks like the `USE_GDAL` flag isn't getting set. I know you fixed that some time back; my fear is that I overwrote something with my recent aggressive changes. Hopefully this was the only thing...
